script/audit_deps.py

```diff
@@ -40,25 +45,22 @@ def main():
     for dir_path, dirs, dummy in os.walk(args.source_root):
         for dir_name in dirs:
             full_path = os.path.join(dir_path, dir_name)
-            skip_dir = False
-            for exclusion in EXCLUDE_PATHS:
-                if full_path.startswith(os.path.join(args.source_root, exclusion)):
-                    skip_dir = True
-                    break
-
-            if not skip_dir:
-                errors += audit_path(full_path, args)
+            errors += audit_path(full_path, args)
 
     return errors > 0
 
 
 def audit_path(path, args):
```

Comment:
I would add a comment to point out that `path` can be either relative or absolute because that confused me as I was reading through this function.

Reply:
Done.